zebra-rpc/Cargo.toml

```
@@ -64,6 +64,8 @@ tracing = "0.1.39"
 hex = { version = "0.4.3", features = ["serde"] }
 serde = { version = "1.0.201", features = ["serde_derive"] }
 
+zcash_primitives = { version = "0.13.0" }
```

```suggestion
zcash_primitives = { version = "0.14.0" }
```

Can we go straight to version 0.14 ?

---

If we bump `zcash_primitives` in `zebra-rpc`, we should also bump it in `zebra-chain` because that's where we impl `zcash_primitives::consensus::Parameters`. However, bumping `zcash_primitives` in `zebra-chain` requires many other changes, so I reverted the bump in [318f945](https://github.com/ZcashFoundation/zebra/pull/8533/commits/318f945d2026a4cdbb8323a30d4b90b99e4de3ab).

---

Yea, that makes sense. Sorry about that, we can update later.